### PR TITLE
fix: translations of CarouselAssetActions

### PR DESCRIPTION
**Pull request by ivan-ha**
### What does it do?

SCR-20221223-poc

when hovering on this button, currently always shows `edit` (the `defaultMessage`) regardless of the language setting, this PR is to fix the issue.

### How to test it?

change to a non-EN language, hover on abovementioned button, you should see the correct text

### Related issue(s)/PR(s)

NIL

### CODE REVIEW

File: packages/core/upload/admin/src/components/MediaLibraryInput/Carousel/CarouselAssetActions.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ export const CarouselAssetActions = ({ asset, onDeleteAsset, onAddAsset, onEditA
       {onEditAsset && (
         <IconButton
           label={formatMessage({
-            id: getTrad('app.utils.edit'),
+            id: 'app.utils.edit',
```
Comment (ronronscelestes) [inline packages/core/upload/admin/src/components/MediaLibraryInput/Carousel/CarouselAssetActions.js:46]: good catch!
I think we should keep the `getTrad` and use the `control-card.edit` translation key though 👀

**Review by ronronscelestes — CHANGES_REQUESTED** [review]

File: packages/core/upload/admin/src/components/MediaLibraryInput/Carousel/CarouselAssetActions.js (hunk 1 above)
Comment (ivan-ha) [reply packages/core/upload/admin/src/components/MediaLibraryInput/Carousel/CarouselAssetActions.js]: sure, updated https://github.com/strapi/strapi/pull/15261/commits/9ed95854ccaf7aac3a0714f7249484cc3b8279ba

**Review by ronronscelestes — APPROVED** [review]
LGTM thank you 💃

**Comment by ivan-ha** [comment]
> codecov/project — 60.49% (-0.01%) compared to 05be584

codecov failed but seems not related, anything else I need to do? @ronronscelestes

**Comment by ronronscelestes** [comment]
> codecov failed but seems not related, anything else I need to do? @ronronscelestes

no we should be good - just waiting for the tests to pass, then I'll merge your PR 😊

**Comment by ivan-ha** [comment]
> no we should be good - just waiting for the tests to pass, then I'll merge your PR 😊

got it, thanks 👍🏻

**Pull request merged**